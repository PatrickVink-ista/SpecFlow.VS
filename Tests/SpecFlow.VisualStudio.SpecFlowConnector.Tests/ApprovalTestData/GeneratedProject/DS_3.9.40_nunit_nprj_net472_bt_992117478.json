{
    "GeneratorOptions": {
        "featureFileCount": 1,
        "scenarioPerFeatureFileCount": 1,
        "newProjectFormat": true,
        "targetFramework": "net472",
        "SpecFlowPackageVersion": "3.9.40"
    }
}
