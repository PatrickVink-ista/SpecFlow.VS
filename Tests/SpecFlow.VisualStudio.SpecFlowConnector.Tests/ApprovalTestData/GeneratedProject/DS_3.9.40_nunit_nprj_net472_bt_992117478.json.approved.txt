{
  "stepDefinitions": [
    {
      "type": "When",
      "regex": "^urna massa \u0022(.*)\u0022 sollicitudin$",
      "method": "LectusDignissimEuFelisSteps.WhenEgetRisusEuAccumsan(String)",
      "paramTypes": "s",
      "expression": "urna massa \u0022(.*)\u0022 sollicitudin",
      "sourceLocation": "#0|10|9|12|10"
    },
    {
      "type": "Given",
      "regex": "^ut et \u0022(.*)\u0022$",
      "method": "LectusDignissimEuFelisSteps.GivenLoremSapienQuisqueVitae(String)",
      "paramTypes": "s",
      "expression": "ut et \u0022(.*)\u0022",
      "sourceLocation": "#0|16|9|18|10"
    },
    {
      "type": "Then",
      "regex": "^molestie augue auctor eu$",
      "method": "LectusDignissimEuFelisSteps.ThenTristiqueAnteLiberoUt(Table)",
      "paramTypes": "st",
      "expression": "molestie augue auctor eu",
      "sourceLocation": "#0|22|9|24|10"
    }
  ],
  "sourceFiles": {
    "0": "<<targetFolder>>\\StepDefinitions\\LectusDignissimEuFelisSteps.cs"
  },
  "typeNames": {},
  "analyticsProperties": {
    "Connector": "<connector>",
    "ImageRuntimeVersion": "v4.0.30319",
    "SFFile": "TechTalk.SpecFlow.dll",
    "SFFileVersion": "3.9.40.46609",
    "SFProductVersion": "3.9.40\u002B11b6bbec89",
    "SourcePaths": "1",
    "StepDefinitions": "3",
    "TargetFramework": ".NETFramework,Version=v4.7.2",
    "TypeNames": "0"
  }
}
